app/screens/GroupMapScreen.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { useGroup } from '../providers/GroupProvider';
+import Map from '../components/Map';
+
+function GroupMapScreen({ navigation }) {
+  const { group } = useGroup();
+
+  // Filter out the members that have their location set, then
+  // map each to an object that the Map component expects.
+  const getMarkers = () => group
```

Comment:
This would be a good case for `useMemo`, which listens for changes to "group".  Otherwise "getMarkers" will be recreated with every render